symfony/flex/1.0/.env.dist

```diff
@@ -0,0 +1,3 @@
+# This file is a "template" of which env vars need to be defined for your application
+# Copy this file to .env file for development, create environment variables when deploying to production
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
```

Comment:
Re-added until symfony/flex#434 is merged. Until then, if you don't have a `.env.dist` file, the "env" configurator won't work - i.e. it won't change your `.env` file when you install things.